src/components/Dashboard/ProprietorPartnersListSm.vue

```diff
@@ -20,46 +20,55 @@
 
         <v-expansion-panel-content>
           <v-list class="pt-0 pb-0">
-            <v-list-item class="email-address-list-item" v-if="party.officer.email">
+            <v-list-item class="email-address-list-item">
               <v-list-item-content>
                 <v-list-item-title class="mb-2 address-title">Email Address</v-list-item-title>
-                <v-list-item-subtitle>{{ party.officer.email }}</v-list-item-subtitle>
+                <v-list-item-subtitle class="email-address-text">
+                  {{ party.officer.email || 'Not Entered'}}
+                </v-list-item-subtitle>
               </v-list-item-content>
             </v-list-item>
 
-            <v-list-item class="delivery-address-list-item" v-if="party.deliveryAddress">
+            <v-list-item class="delivery-address-list-item">
               <v-list-item-content>
                 <v-list-item-title class="mb-2 address-title">Delivery Address</v-list-item-title>
                 <v-list-item-subtitle>
-                  <ul class="address-subtitle pre-line">
+                  <ul class="address-subtitle pre-line" v-if="party.deliveryAddress">
```

Comment:
I just realized that the ticket was not clear -- there may be no proprietor/partners at all and that's when "Not Entered" should display in this component.

Reply:
^^ I added a ticket comment to ask Linda for clarification

Reply:
When there is no "proprietor/partners" or delivery/mailing address at all, it displays "Complete your filing to display".

Reply:
Right.

Note that in both scenarios there may be no parties (orgpersons). You need to differentiate between those scenarios in your code. I assume you can check if the current entity is using a temp registration number (ie, it's still a registration) or is using an actual business number (ie, it's now a firm).

Reply:
![image](https://user-images.githubusercontent.com/51458187/167209731-81f82876-2609-4f1a-beef-3dbdc0a8a879.png)

The current implementation from line 3 to 5 (before this branch) covers the scenario for no parties, and it displays "Complete your filing to display".

Reply:
I'm resolving this conversation now, but this code will need to be re-reviewed when we better understand what "missing data" scenarios we might encounter, and then we can look at what messages are displayed in each case.